src/main/java/com/lesson7/solution/service/impl/TodoServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package com.lesson7.solution.service.impl;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.lesson7.solution.entity.TaskHistory;
+import com.lesson7.solution.entity.Todo;
+import com.lesson7.solution.exception.EntityNotFoundException;
+import com.lesson7.solution.mapper.TodoMapper;
+import com.lesson7.solution.model.TodoCreateDto;
+import com.lesson7.solution.model.TodoResponseDto;
+import com.lesson7.solution.model.TodoUpdateDto;
+import com.lesson7.solution.repository.TaskHistoryRepository;
+import com.lesson7.solution.repository.TodoRepository;
+import com.lesson7.solution.service.TodoService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@RequiredArgsConstructor
+@Service
+public class TodoServiceImpl implements TodoService {
+    private final TodoRepository todoRepository;
+    private final TaskHistoryRepository taskHistoryRepository;
+    private final TodoMapper todoMapper;
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
`ObjectMapper objectMapper` also could be autowired
```suggestion
    private final ObjectMapper objectMapper;
```